features/support/server.js

```diff
@@ -0,0 +1,8 @@
+require('http')
```

Comment:
Isn't this file created in the steps?

> `And a file named "server.js" with:`

Why do we need it here?

Reply:
I thought the TCP feature file uses it, but it might not be the case after all, when I'm looking at it now. I'll check.